CONTRIBUTING.md

```diff
@@ -17,6 +17,12 @@ brew install orbstack
 ```
 or [OrbStack Website](https://orbstack.dev/)
 
+We use [dbmate](https://github.com/amacneil/dbmate) to manage migrations, so please install that if
+you don't have it already:
+```
+brew install dbmate
```

Comment:
This is included in Hermit, so remove this bit.

Reply:
Ahh nice! Removed